src/styles/card.css

```diff
@@ -0,0 +1,48 @@
+.card {
+  width: 198px;
```

Comment:
I would add padding to the block element (the parent or the container/wrapper) and this way most of the children won't need extra code (for ex. margin, padding, position, etc...). 
Plus if the padding of the card will be changed later, we would only need to modify in one place.

As you can see in the screenshot of Figma - https://imgur.com/a/TXpmIh4 the card element has inner elements that has distances from the card element and can help you write the code.